packages/ui/src/working-groups/components/WorkingGroupsList.tsx

```diff
@@ -34,14 +36,18 @@ export const WorkingGroupsList = ({ groups }: WorkingGroupsListProps) => {
 }
 
 const HeaderColumnWrapper = styled.div`
-  display: flex;
-  justify-content: flex-end;
+  display: grid;
+  grid-template-columns: 108px repeat(2, minmax(400px, 1fr)) 40px;
+  grid-template-rows: 1fr;
+  grid-column-gap: 24px;
 `
 
 const ElementsWrapper = styled.div`
-  display: flex;
-  justify-content: space-around;
-  width: 50%;
+  display: grid;
+  grid-template-columns: 64px 116px 30px 140px;
```

Comment:
Same here but also please increase the column size a bit to: `64px 140px 64px 140px`